src/msg/async/rdma/Infiniband.cc

```diff
@@ -126,8 +126,8 @@ Device::Device(CephContext *cct, ibv_device* d, struct ibv_context *dc)
     lderr(cct) << __func__ << " open rdma device failed. " << cpp_strerror(errno) << dendl;
     ceph_abort();
   }
-  int r = ibv_query_device(ctxt, device_attr);
-  if (r == -1) {
+  int rst = ibv_query_device(ctxt, device_attr);
```

Comment:
nit, `rst` is confusing. it could be `reset` or `result`. i'd suggest keep the name as it is.

Reply:
@tchaikov Agree. Thanks for the suggestion.